chain/chain/src/store.rs

```diff
@@ -2007,15 +2008,17 @@ impl<'a> ChainStoreUpdate<'a> {
             GCMode::Fork(tries) => {
                 // If the block is on a fork, we delete the state that's the result of applying this block
                 for shard_id in 0..header.chunk_mask().len() as ShardId {
+                    // TODO: pass in the actual shard version that this block uses
```

Comment:
is this already done?

Reply:
No, this is not. I will change this later.

Reply:
Could you create an issue for that and add the issue number in the comment?

Reply:
https://github.com/near/nearcore/issues/4710